plugin.video.ispot/default.py

```diff
@@ -0,0 +1,335 @@
+#   Copyright (C) 2017 Lunatixz
+#
+#
+# This file is part of iSpot.
+#
+# iSpot is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# iSpot is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with iSpot.  If not, see <http://www.gnu.org/licenses/>.
+
+# -*- coding: utf-8 -*-
+import os, re, sys, time, datetime, traceback
+import urllib, urllib2, base64, HTMLParser, socket
+import xbmc, xbmcgui, xbmcplugin, xbmcvfs, xbmcaddon
+
+from simplecache import use_cache, SimpleCache
+
+if sys.version_info < (2, 7):
+    import simplejson as json
+else:
+    import json
+
+## GLOBALS ##
+baseurl='https://www.ispot.tv/'
+TIMEOUT = 15
+
+# Plugin Info
+ADDON_ID = 'plugin.video.ispot'
+REAL_SETTINGS = xbmcaddon.Addon(id=ADDON_ID)
```

Comment:
I mean the id in the Addon constructor. xbmcaddon.Addon() is enough. You only need the id if you're trying to access a different plugin